src/main/kotlin/no/nav/familie/ef/søknad/mapper/SøknadMapper.kt

```diff
@@ -111,53 +108,21 @@ object SøknadMapper {
-                                                                    true),
-                                                        dokumentfelt("Avtale om samvær"),
-                                                        Søknadsfelt("Har den andre forelderen samvær med barnet",
-                                                                    "Ja, men ikke mer enn vanlig samværsrett"),
-                                                        Søknadsfelt("Har dere skriftlig samværsavtale for barnet?",
-                                                                    "Ja, men den beskriver ikke når barnet er sammen med hver av foreldrene"),
-                                                        dokumentfelt("Avtale om samvær"),
-                                                        Søknadsfelt("Hvordan praktiserer dere samværet?",
-                                                                    "Litt hver for oss"),
-                                                        Søknadsfelt("Bor du og den andre forelderen til [barnets navn] i samme hus/blokk, gårdstun, kvartal eller vei?",
-                                                                    true),
-                                                        Søknadsfelt("Har du bodd sammen med den andre forelderen til [barnets fornavn] før?",
-                                                                    true),
-                                                        Søknadsfelt("Når flyttet dere fra hverandre?",
-                                                                    LocalDate.of(2018, 7, 21)),
-                                                        dokumentfelt("Erklæring om samlivsbrudd"),
-                                                        Søknadsfelt("Hvor mye er du sammen med den andre forelderen til barnet?",
-                                                                    "Vi møtes også uten at barnet er til stede"),
-                                                        Søknadsfelt("Beskriv  hvor mye er du sammen med den andre forelderen til barnet?",
-                                                                    "Vi sees stadig vekk"))))
-    }
-
-    @Suppress("LongLine")
-    private fun folkeregisterbarn(): Folkeregisterbarn {
-        return Folkeregisterbarn(Søknadsfelt("Navn", "Lykkeliten"),
-                                 Søknadsfelt("Fødselsnummer", Fødselsnummer("31081953069")),
-                                 Søknadsfelt("Har samme adresse som søker", true),
-                                 Søknadsfelt("Barnets andre forelder",
-                                             Forelder(person = Søknadsfelt("personalia", personMinimum()),
-                                                      adresse = adresseSøknadsfelt())),
-                                 Søknadsfelt("samvær",
+    private fun kommendeBarn(): NyttBarn {
```

Comment:
pirk, men kanskje disse funksjonene kan bli navngitt etter de nye navnene, i stedet for de gamle? (`fun kommendebarn `og `fun folkeregistrertbarn`)